packages/amplify-util-mock/src/CFNParser/intrinsic-functions.ts

```diff
@@ -51,7 +51,7 @@ function templateReplace(template: string, args: any = {}) {
   });
 }
 
-export function cfnGetAtt(valNode, { resources }: CloudFormationParseContext, processValue) {
+export function cfnGetAtt(valNode, { resources }: CloudFormationParseContext) {
```

Comment:
this and similar methods in CFNParser are registered as event listeners. They're compatible as long as they have less or equal number of params.